### ISSUE TITLE
Multiple Ambisonic Sources not playing in sync

### ISSUE THREAD

**Issue by riiight** [RenderHeads/UnityPlugin-AVProMovieCapture#239]
I have a simple 3D Unity project with 4 objects with Ambisonic Source components that are each playing music clips that should be in sync. When I turn off the Ambisonic Sources and just have the Audio Sources and then hit "Play", they play in sync as expected. However, when I turn on the Ambisonic Sources on the 4 objects, they start getting out of sync. Some of the clips play slightly faster than others.

Can you help with this?

**Comment by riiight** [comment]
After I closed some services on my machine it seems to be working ok.

**Comment by Chris-RH** [comment]
Hi.

Which platform are you using?
What are the specs of your device?
SRP/URP/HDRP?
Have you tried the Ambisonic demo scene and do you get similar results?
Is there anything else running that may be interfering?

**Comment by riiight** [comment]
I am referring to playing directly in Unity. I haven't gotten to the point where I have compiled it for any devices. 
SRP
Yes, I tried the demo and it works fine. The issue I have is about syncing 2+ audio clips that need to be played in unison which the demo does not have. 
I've shut down everything I can.

That all said, I think the issue may be related to inherent Unity audio latency issues (though, as I mentioned, the Ambisonic Sources make it noticeably worse) so I will close this comment for now.

Thanks for your help.
